Jellyfin.Server/Extensions/ApiServiceCollectionExtensions.cs

```diff
@@ -71,6 +71,10 @@ public static IMvcBuilder AddJellyfinApi(this IServiceCollection serviceCollecti
                 // Clear app parts to avoid other assemblies being picked up
                 .ConfigureApplicationPartManager(a => a.ApplicationParts.Clear())
                 .AddApplicationPart(typeof(StartupController).Assembly)
+                .AddJsonOptions(options => 
+                {
+                    options.JsonSerializerOptions.PropertyNamingPolicy = null;
```

Comment:
what does this do?

Reply:
This line is what actually fixes the issue. The changes to the property names in the other file were just for clarity regarding what the endpoint expects. The default JSON options use a the camel case property naming policy, so when we return `StartupCollectionDto`, it takes every property of that object and camel cases it. Setting the naming policy to null leaves the property names unchanged.

Reply:
Hmm, this would make an excellent comment right there.

Reply:
Good point. Added!